modules/client-configuration/pages/contact-methods-traditional.adoc

```diff
@@ -3,14 +3,46 @@
 
 Traditional clients can communicate with the {productname} Server using a range of methods.
 
-The {productname} daemon ([command]``rhnsd``) runs on traditional client systems and periodically connects with {productname} to check for new updates and notifications.
+The lightweight {productname} daemon ([command]``rhnsd``) runs on traditional client systems.
+The daemon periodically connects with {productname} to check for new updates and notifications.
+
+[WARNING]
+====
+The [command]``rhnsd`` method has serious limitations.
+It does not scale as well as Salt.
+Newer operating systems are not supported and will not be added in the future.
+====
+
+For more information, see xref:client-configuration:contact-methods-rhnsd.adoc[].
+
+Traditional with OSAD::
+is the same as traditional but allows the server to
+push updates to clients.
+OSAD is an alternative contact method between {productname} and traditional clients.
```

Comment:
I would not say "alternative". osad enhances rhnsd

Reply:
ok